assets/src/stories-editor/helpers/index.js

```diff
@@ -9,6 +9,7 @@ import memize from 'memize';
 /**
  * WordPress dependencies
  */
+import '@wordpress/core-data';
```

Comment:
I don't understand this. What is the current problem and how does this fix it?

Reply:
The JS unit tests were failing without this change because the `core` data store was not initialized. See #3123 for the failed build info.

It is used referenced here:

https://github.com/ampproject/amp-wp/blob/e33ce8787ea08851843a2bfdc2abb600dfe43023/assets/src/stories-editor/helpers/index.js#L81

Because `dispatch( 'core' )` was `undefined`, this causes errors.

Now there were two options:

1. Move `const { saveMedia } = dispatch( 'core' );` to within `uploadVideoFrame`, the only place where it is used.
2. Import `@wordpress/core-data` so the data store is initialized in the JS unit tests

Where option 2 felt more robust to me.

Not sure why exactly this suddenly happened though. Perhaps it was the change at https://github.com/WordPress/gutenberg/pull/16932.